486_Project/Text_Game.py

```diff
@@ -57,22 +64,118 @@ def select_class():
 room5 = Room("""
 mean guy first
 """)
+room6 = Room("""
+mean dog first
+""")
+
+# creating rooms and their descriptions
+Beginning_Room = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Animal_Starting_Room = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Wolf_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Nest_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Giant_Eagle_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Dead_Man_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Giant_Lizard_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Human_Starting_Room = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Corrupted_Swordsman_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Corrupted_Swordsman_Encounter_2 = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Corrupted_Acolyte_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Convenient_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+New_Weapon_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Monster_Starting_Room = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Bugbear_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Goblin_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Sleeping_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Big_Event = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Death_Dog_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Dragon_Intro = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Dragon_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
```

Comment:
Great that these have proper titles now! They are far easier to distinguish between because of this.